frontend/src/containers/Homepage/Homepage.js

```diff
@@ -22,7 +22,7 @@ const Homepage = ({ location, user, setJwt }) => {
     <div className="homepage container-fluid ">
       <div className="row">
         <div className="header">
-          <Header user={user} setJwt={setJwt} />
+          <Header user={user} setJwt={setJwt} scrollDown={executeScroll} />
```

Comment:
Maybe a more descriptive name would be appropriate here, like `scrollToRecipes`.